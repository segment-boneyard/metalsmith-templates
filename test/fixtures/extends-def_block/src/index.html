---
title: TITLE
content: CONTENT
extends: base.html
---
{{ content }} here
