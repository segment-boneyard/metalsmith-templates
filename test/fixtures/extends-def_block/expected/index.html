<html>
<head>
<title>TITLE</title>
<style>

</style>
</head>
<body>
CONTENT here

</body>
</html>
